infra/Pos/Slotting/MemState/Class.hs

```diff
@@ -8,24 +8,38 @@ import           Control.Monad.Trans (MonadTrans)
 import           Universum
 
 import           Pos.Core.Types      (EpochIndex, Timestamp)
-import           Pos.Slotting.Types  (SlottingData)
+import           Pos.Slotting.Types  (EpochSlottingData)
 
 -- | 'MonadSlotsData' provides access to data necessary for slotting to work.
 class Monad m => MonadSlotsData m where
 
     getSystemStart :: m Timestamp
 
-    getSlottingData :: m SlottingData
+    getEpochLastIndex :: m EpochIndex
 
-    waitPenultEpochEquals :: EpochIndex -> m ()
+    -- TODO(ks): Maybe add other usefull functions directly?
+    -- getEpochPenultIndex :: m EpochIndex
```

Comment:
I think this is not needed as I wrote in a standalone comment to this PR (i. e. we should completely deprecated everything `penult` :)).